package.json

```diff
@@ -4,12 +4,13 @@
   "description": "Google Labs repo for rapid prototyping",
   "type": "module",
   "scripts": {
-    "sync": "npm i && npm run clean:build && npm run build",
+    "clean": "git clean -fdx",
```

Comment:
How about `git clean -Xdf -e .vscode/settings.json`

Using `-X` means it will only delete files that are gitignored. This is a bit safer than `-x`, since that way if somebody has a new file they forgot to `git add`, it won't get removed by `clean`.

Also adding `-e .vscode/settings.json` is useful because usually folks will want to keep any of their local settings around, even though we don't check them into Git.

Reply:
```suggestion
    "clean": "git clean -Xdf -e .vscode/settings.json",
```